ddtrace/appsec/_remoteconfiguration.py

```diff
@@ -21,8 +21,15 @@
 log = get_logger(__name__)
 
 
-def enable_appsec_rc(tracer):
-    # type: (Tracer) -> None
+def disable_appsec_rc():
+    log.warning("Disabling remote configuration")
+    RemoteConfig.disable()
```

Comment:
Wouldn't this disable the RCM client for every other product too? 🤔

Reply:
Good point!